lldb/test/API/lang/swift/variables/consume_operator/TestSwiftConsumeOperator.py

```diff
@@ -71,324 +71,301 @@ def get_var(self, name):
         return frame.FindVariable(name)
 
     def do_check_copyable_value_test(self):
-        # We haven't defined varK yet.
-        varK = self.get_var('k')
-
-        self.assertIsNone(varK.value, "varK initialized too early?!")
+        # We haven't defined k yet.
+        self.assertIsNone(self.get_var('k').value, "k initialized too early?!")
 
         # Go to break point 2. k should be valid.
         self.process.Continue()
-        self.assertGreater(varK.unsigned, 0, "varK not initialized?!")
+        self.assertGreater(self.get_var('k').unsigned, 0, "k not initialized?!")
```

Comment:
@gottesmm FYI, I don't think there is any guarantee that varK remains valid after process.Continue().